chromium_src/tools/grit/grit/gather/policy_json.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The Brave Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import override_utils
+
+@override_utils.override_method(PolicyJson)
+def SetDefines(self, original_method, defines):
+  if '_google_chrome' in defines:
```

Comment:
I wonder if we need this check at all.

Reply:
Resolved in d9154e9649727764dddc315cd8922c27361f6f90.